### PR TITLE
Update README for instructions to start.

### PR DESCRIPTION
**Pull request by dannywillems**
Add links to demo and remove an useless section.
Add also a link to the manual (as asked on Twitter).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +21,7 @@ Eliom). It provides modules for
 * displaying tips, and
 * easily sending notifications to the users.
 
-Ocsigen Start is in an early stage of development. More modules will
-be added and more customizability.
+More modules will be added and more customizability.
```
Comment (vasilisp) [inline README.md]: This sentence looks out of place now. I would suggest removing it altogether.

File: README.md [hunk 2]
```diff
@@ -47,8 +47,7 @@ opam install ocsigen-start
 eliom-distillery -name myproject -template os.pgocaml
 ```
 
-###<a id="create-your-database"></a>Create your database
-Have look at the file README in your directory.
-It explains how to configure your database.
+And take a look to the generated README.md to start.
```
Comment (vasilisp) [inline README.md]: The grammar and structure is a bit off. I suggest

> To _get started_, take a look _at_ the generated `README.md`.

File: README.md (hunk 1 above)
Comment (dannywillems) [inline README.md]: Done @vasilisp

**Comment by dannywillems** [comment]
@vasilisp Done.

**Comment by vasilisp** [comment]
Thanks!

**Pull request merged**